tck/src/test/java/org/reactivestreams/tck/BrokenExampleBlackboxTest.java

```diff
@@ -0,0 +1,80 @@
+/************************************************************************
+ * Licensed under Public Domain (CC0)                                    *
+ *                                                                       *
+ * To the extent possible under law, the person who associated CC0 with  *
+ * this code has waived all copyright and related or neighboring         *
+ * rights to this code.                                                  *
+ *                                                                       *
+ * You should have received a copy of the CC0 legalcode along with this  *
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.*
+ ************************************************************************/
+
+package org.reactivestreams.tck;
+
+import org.reactivestreams.Publisher;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+import org.reactivestreams.example.unicast.AsyncIterablePublisher;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Collections;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+@Test
+public class BrokenExampleBlackboxTest extends SubscriberBlackboxVerification<Integer> {
```

Comment:
@ktoso What is the purpose of the "Broken" tests?

Reply:
Oh good catch... I think this leaked from some working branch where I used those to break rules to see if they fail nicely -- removing both.